storage/src/test/java/org/apache/kafka/storage/internals/log/LogSegmentTest.java

```diff
@@ -103,6 +103,16 @@ private MemoryRecords records(long offset, String... records) {
             Compression.NONE, TimestampType.CREATE_TIME, simpleRecords.toArray(new SimpleRecord[0]));
     }
 
+    private MemoryRecords makeBatch(long offset, String... records) {
```

Comment:
Sorry, didn't realize there is another method with the same signature. It would be useful to make them consistent. How about `v1Records` and `v2Records`?

Reply:
It looks like there's a `records(offset, size)` function that creates a singular record batch with `size` records. It looks like I can just reuse that function and rename the two `records` functions to `v1/v2Records` for clarity.

Reply:
It looks like the other `records(offset, size)` method signature being used in `testAppendFromFile` is slightly misleading. `size` is not actually the number of records to add in a batch, but rather the size of a single record for a specified offset. I'll still rename `records(offset, records...)` and `makeBatch` to `v1Records` and `v2Records`